hw/ip/edn/data/edn.hjson

```diff
@@ -0,0 +1,264 @@
+    },
+    { name: "EndPointBusWidth3",
+      desc: "Power of two width of the obsfucated entropy data bus for end point 3",
+      type: "int",
+      default: "8",
+      local: "true"
+    },
+  ],
+  interrupt_list: [
+    { name: "edn_cmd_req_done"
+      desc: "Asserted when a software csrng request has completed."}
+    { name: "edn_fifo_err"
+      desc: "Asserted when a FIFO error occurs."}
+  ],
+  inter_signal_list: [
+    { struct:  "edn",  // edn_req_t, edn_rsp_t
+      type:    "req_rsp",
+      name:    "edn_cmd",
+      act:     "rsp",
+      width:   "4",
+      package: "edn_pkg",
+    }
+  ],
+
+  regwidth: "32",
+  registers: [
+    { name: "REGEN",
+      desc: "Register write enable for all control registers",
+      swaccess: "rw1c",
+      hwaccess: "hro",
+      fields: [
+        {
+            bits:   "0",
+            desc: ''' When true, the EDN_CONF, EDN_RCT_HEALTH, EDN_APT_HEALTH and EDN_SEED registers can be modified.
+            When false, they become read-only. Defaults true, write one to clear. Note that this needs to be
+            cleared after initial configuration at boot in order to lock in the listed register settings.
+            '''
+            resval: 1
+        },
+      ]
+    },
+    { name: "CTRL",
+      desc: "EDN control register",
+      swaccess: "rw",
+      hwaccess: "hro",
+      fields: [ 
+        {
+            bits: "0",
+            name: "EDN_ENABLE",
+            desc: '''
+                  Setting this bit will enable the EDS module. 
+                  '''
+        },
+        {
+            bits: "1",
+            name: "BCAST_MODE_DIS",
+            desc: '''
+                  Setting this bit will disable the EDN block to self ack the ENDPOINT0 entropy request
+                  bus. This mode for ENDPOINT0 will ignore the request input.
+                  '''
+        },
+        {
+            bits: "2",
+            name: "AUTO_REQ_MODE",
+            desc: '''
+                  Setting this bit will enable the EDN block to automatically send another request to 
+                  CSRNG application interface. It is assuming that the command has been set up ahead of
+                  time, such that when the previous command has returned ack, then a new command will be
+                  send out again without software intervention. It is expected that the generate command
+                  will be repeatedly so that a continuous supply of entropy can be delivered to the
+                  end points.
+                  '''
+        },
+        {
+            bits: "3",
+            name: "BOOT_REQ_DIS",
+            desc: '''
+                  Setting this bit will disable the feature where the EDN block will automatically
+                  send a boot time request to the CSRNG application interface.
+                  The purpose of this feature is to request entropy as fast as possible after reset,
+                  and during chip boot time.
```

Comment:
sorry i'm a bit confused if this is supposed to be set or cleared for the boot scenario.

Reply:
ooo okay i see in the programmers model now.